tests/delta.test

```diff
@@ -33,9 +33,15 @@ for delta in $inputdir/*.delta
 do
     for inbuf in $bufsizes
     do
-        cmd="$bindir/rdiff -I$inbuf patch /dev/null $delta $new"
+        cmd="$bindir/rdiff -I$inbuf -f patch /dev/null $delta $new"
         run_test $cmd
         expect=$inputdir/`basename $delta .delta`.expect
 	check_compare $expect $new
     done
 done
+
+# Make sure rdiff can't overwrite files if ran without -f
+# RS_IO_ERROR == 100 should be returned if file exists
+cmd="$bindir/rdiff patch /dev/null $inputdir/01.delta $new"
+echo "    $cmd" >&2
+$cmd 2>&1 | grep -q 'File exists' || fail_test "$?" "File exists not returned when overwriting files without --force"
```

Comment:
Added test that should print proper error when file exists and no --force was given.